app/src/main/kotlin/com/intive/picover/parties/di/PartiesModule.kt

```diff
@@ -0,0 +1,20 @@
+package com.intive.picover.parties.di
+
+import cafe.adriel.voyager.hilt.ScreenModelFactory
+import cafe.adriel.voyager.hilt.ScreenModelFactoryKey
+import com.intive.picover.parties.viewmodel.PartyDetailsViewModel
+import dagger.Binds
+import dagger.Module
+import dagger.hilt.InstallIn
+import dagger.hilt.android.components.ActivityComponent
+import dagger.multibindings.IntoMap
+
+@Module
+@InstallIn(ActivityComponent::class)
+abstract class PartiesModule {
+
+	@Binds
+	@IntoMap
+	@ScreenModelFactoryKey(PartyDetailsViewModel.Factory::class)
+	abstract fun bindPartyDetailsViewModelFactory(factory: PartyDetailsViewModel.Factory): ScreenModelFactory
```

Comment:
Internally there's a map of ScreenModelFactory in voyager, if we have a view model with arguments we need to create an AssistedFactory for it, and bind this factory into that internal map. Perhaps in the future we could write some custom KSP processor to generate that module for us

Reply:
Is it possibile to have default binding for `SavedStateHandle` in view model?

Reply:
There will be no `SavedStateHandle` 😄

Reply:
Oh... right 😄  I'm still tied to android world 😄